modules/ossm-release-2-4-8.adoc

```diff
@@ -0,0 +1,34 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-release-2-5-2.adoc
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-release-2-4-8_{context}"]
+= {SMProductName} version 2.4.8
+
+//Release is scheduled for May 22, 2024.
+//As of May 8, there are no new features so the phrase "new features" has been removed. This is a z-stream release to update containers before they are Grade B or C on May 28.
+//Includes 2.5.2, 2.4.8, 2.3.12
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.12 and later.
+
+The most current version of the {SMProductName} Operator can be used with all supported versions of {SMProductShortName}. The version of {SMProductShortName} is specified using the `ServiceMeshControlPlane`.
+
+[id="ossm-component-updates-2-4-8_{context}"]
+== Component updates
+
+// Release is scheduled for May 22, 2024. Code and Doc Freeze is scheduled for May 10, 2024. Component versions should be available after May 10.
```

Comment:
Can this old note be removed or updated?

Reply:
It's for historical purposes for management.